src/Lykke.Job.BlockchainCashinDetector/Workflow/Sagas/CashinSaga.cs

```diff
@@ -274,6 +280,32 @@ public class CashinSaga
             }
         }
 
+        [UsedImplicitly]
+        private async Task Handle(DepositBalanceDetectionsDeduplicationLockUpdatedEvent evt, ICommandSender sender)
+        {
+            _log.WriteInfo(nameof(DepositBalanceDetectionsDeduplicationLockUpdatedEvent), evt, "");
+
+            try
+            {
+                var aggregate = await _cashinRepository.TryGetAsync(evt.OperationId);
```

Comment:
You should use GetAsync, not TryGetAsync, since this event can be handled only for existing cashin